src/main/java/dev/walshy/sfmetrics/MetricsModule.java

```diff
@@ -40,8 +41,9 @@ public static void start() {
         metrics.addCustomChart(new CommandChart());
         metrics.addCustomChart(new ServerSizeChart());
         metrics.addCustomChart(new CompatibilityModeChart());
-        metrics.addCustomChart(new MetricsVersionChart());
 
-        SlimefunPlugin.instance().getLogger().info("Now running MetricsModule v" + VERSION);
+        // Metric specific
+        metrics.addCustomChart(new MetricsVersionChart());
+        metrics.addCustomChart(new MetricAutoUpdatesChart());
```

Comment:
You forgot to update the class name in this class...

Reply:
Give me a minute! I just did this with nano